qtfred/src/mission/dialogs/CustomWingNamesDialogModel.cpp

```diff
@@ -13,6 +13,18 @@ CustomWingNamesDialogModel::CustomWingNamesDialogModel(QObject * parent, EditorV
 
 bool CustomWingNamesDialogModel::apply() {
 	int i;
+	for (i = 0; i < MAX_STARTING_WINGS; i++) {
```

Comment:
clang-tidy will probably tell you that this (and the loops below) could be converted to for-each loops.